docs/contributing/coding-guidelines/ros-nodes/class-design.md

```diff
@@ -1,5 +1,68 @@
 # Class design
 
+We'll use the `autoware_gnss_poser` package as an example.
+
+## Namespaces
+
+```cpp
+namespace autoware::gnss_poser
+{
+...
+} // namespace autoware::gnss_poser
+```
+
+- Everything should be under `autoware::gnss_poser` namespace.
+- Closing braces should contain a comment with the namespace name.
```

Comment:
I´d use "must" instead of "should", in any case, `cpplint` will complain if there isn't a comment

Reply:
Done with https://github.com/autowarefoundation/autoware-documentation/pull/547/commits/fea457f9918457fb8c5eb22adb2ec92ee89e8928